addons/mail/__manifest__.py

```diff
@@ -254,15 +253,19 @@
             'web/static/src/core/**/*',
             'web/static/src/env.js',
             ('remove', 'web/static/src/legacy/js/services/ajax_service.js'),
-            ('remove', 'web/static/src/legacy/js/common_env.js'),
             ('remove', 'web/static/src/legacy/js/widgets/translation_dialog.js'),
-            ('remove', 'web_tour/static/src/js/tour_manager.js'),
-            ('remove', 'web_tour/static/src/js/tour_service.js'),
             'web/static/src/views/fields/formatters.js',
             'web/static/src/views/fields/file_handler.*',
             'web/static/src/views/fields/many2many_tags/tags_list.*',
 
             'mail/static/src/new/**/*',
+            'web/static/src/legacy/js/core/misc.js',
+            'web/static/src/legacy/js/env.js',
+            'web/static/src/legacy/js/owl_compatibility.js',
+            'web/static/src/legacy/js/services/data_manager.js',
+            'web/static/src/legacy/js/services/session.js',
+            'web/static/src/legacy/legacy_load_views.js',
+            'web/static/src/legacy/utils.js',
```

Comment:
Are all of those introduced because the legacy env is added?

Reply:
it's not only because of the env but for the tour. In fact, most of them are introduced due to the tour